### PR TITLE
feat: Add homepage, recruitment page, components, and customizations

### PR DESCRIPTION
**Pull request by Alizaaaja4**
### What does this PR do?
- Adds the `HomePage` and `RecruitmentPage` with their respective layouts and styles.
- Implements reusable components:
  - Navbar
  - Footer
  - ContentHome
  - ContentRecruitment
  - CarouselDivisi
  - CarouselEvent
- Customizes Tailwind CSS in `tailwind.config.js` to include new color schemes.
- Adds several assets (images and icons) to the `/public` folder for the website.

### Screenshots
Attached screenshots of the pages and components for reference.

### Checklist:
- [x] HomePage layout and content
- [x] RecruitmentPage layout and content
- [x] Navbar and Footer components
- [x] ContentHome and ContentRecruitment sections
- [x] CarouselDivisi and CarouselEvent
- [x] Tailwind CSS customization
- [x] New assets in `/public`

### Documentation
Screenshots of the work have been attached as a reference.
image
image
image
image
image
image

### CODE REVIEW

**Comment by DiyArc-350** [comment]
overall LGTM

File: components/navbar.tsx
Comment (DiyArc-350) [inline components/navbar.tsx:1]: kinda nitpick here, i think the vertical line next to the logo need a little bit of padding

File: public/daskom icon.svg
Comment (DiyArc-350) [inline public/daskom icon.svg:1]: oh, i see

Comment (Alizaaaja4) [reply public/daskom icon.svg:1]: Actually, the lines and logo have been combined into one image by the UI/UX team. However, I would ask them to revise it to be more flexible, for example separating lines as separate elements that can be managed with CSS.

Comment (DiyArc-350) [reply public/daskom icon.svg:1]: okaay

**Review by Zendin110206 — COMMENTED** [review]
I'm sorry. Is there a margin for the footer? It seems like there's a left and right margin, and I think it would be better if there were no margins (just my opinion). Overall, looks good to me

**Comment by Soedipo** [comment]
> I'm sorry. Is there a margin for the footer? It seems like there's a left and right margin, and I think it would be better if there were no margins (just my opinion). Overall, looks good to me

I aggree with this, not only at footer but at the whole screen, make it zero for the margin so it has no whitespace at the sides

File: app/layout.tsx
Comment (Soedipo) [inline app/layout.tsx:1]: image

**Review by Soedipo — CHANGES_REQUESTED** [review]
> I'm sorry. Is there a margin for the footer? It seems like there's a left and right margin, and I think it would be better if there were no margins (just my opinion). Overall, looks good to me - @Zendin110206 

I aggree with this, not only at footer but at the whole screen, make it zero for the margin so it has no whitespace at all sides

File: app/layout.tsx
Comment (Alizaaaja4) [reply app/layout.tsx:1]: I've applied the revision to remove the margin across the entire screen. Now there’s no whitespace at all sides. Please see the updated screenshot below for confirmation:

image

Comment (Soedipo) [reply app/layout.tsx:1]: Okkkkk LGTM I'll approve this 😁👌

**Review by Soedipo — APPROVED** [review]

**Comment by Soedipo** [comment]
I want to give advice for front-end team,, when PR, if there's animation or dynamic styling at your requirenment/works/PR please provide the video or maybe a GIF so we all could check it too. 
Good luck ^_^

**Comment by Alizaaaja4** [comment]
> I want to give advice for front-end team,, when PR, if there's animation or dynamic styling at your requirenment/works/PR please provide the video or maybe a GIF so we all could check it too.
> Good luck ^_^

Thank you for the suggestion! For the next PR, I will try using a GIF to showcase animations or dynamic styling to make it easier for the review 👍

**Pull request merged**